lib/node_modules/@stdlib/stats/base/dists/planck/median/benchmark/benchmark.native.js

```diff
@@ -46,10 +46,9 @@ bench( pkg+'::native', opts, function benchmark( b ) {
 	var i;
 
 	len = 100;
-	lambda = new Float64Array( len );
-	for ( i = 0; i < len; i++ ) {
-		lambda[ i ] = uniform( 1.0, 10.0 );
-	}
+	
+	lambda = uniform(len, 0.1, 10.0);
```

Comment:
```suggestion
	lambda = uniform( 100, 0.1, 10.0 );
```
as per the suggested changes the `var len` should be removed as the original implementation worked just fine.
Could you pls do the same for the `benchmark.js` file as well.

Reply:
Just adding on to it, once the `len` variable is removed, the initialization of `y` in the benchmark loop will be updated to:  

```js
y = median( lambda[ i % lambda.length ] );
```

Basically, everything that was done in the original PR, as Neeraj already pointed out.